pin_dic:
  1: 'servo'
  2: 'servo'
  3: 'servo'
  4: 'servo'
  5: 'servo'
  6: 'servo'
  7: 'servo'
  8: 'servo'
  9: 'servo'
  10: 'motor'
  11: 'motor'
  12: 'motor'
  13: 'motor'
  14: 'servo'
  15: 'servo'
  16: 'servo'
